pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -304,7 +304,7 @@ func SupportsQuotas(m mount.Interface, path string) (bool, error) {
 // AssignQuota chooses the quota ID based on the pod UID and path.
 // If the pod UID is identical to another one known, it may (but presently
 // doesn't) choose the same quota ID as other volumes in the pod.
-func AssignQuota(m mount.Interface, path string, poduid types.UID, bytes *resource.Quantity) error { //nolint:staticcheck // SA4009 poduid is overwritten by design, see comment below
+func AssignQuota(m mount.Interface, path string, _ types.UID, bytes *resource.Quantity) error {
```

Comment:
Please remove `poduid types.UID` parameter, the function is called only on few places.

Reply:
Updated.